test/protocol/clients/BosonVoucherTest.js

```diff
@@ -2349,6 +2431,59 @@ describe("IBosonVoucher", function () {
     });
   });
 
+  context("callExternalContract()", function () {
+    let mockSimpleContract, calldata;
+
+    beforeEach(async function () {
+      // Deploy a random contract
+      const MockSimpleContract = await ethers.getContractFactory("MockSimpleContract");
+      mockSimpleContract = await MockSimpleContract.deploy();
+      await mockSimpleContract.deployed();
+
+      // Generate calldata
+      calldata = mockSimpleContract.interface.encodeFunctionData("testEvent");
+    });
+
+    it("Should call external contract and emit its events", async function () {
+      const tx = await bosonVoucher.connect(assistant).callExternalContract(mockSimpleContract.address, calldata);
+
+      const receipt = await tx.wait();
+      const event = getEvent(receipt, mockSimpleContract, "TestEvent");
+
+      assert.equal(event._value.toString(), "1");
+    });
+
+    context("💔 Revert Reasons", async function () {
+      it("_to is the zero address", async function () {
+        await expect(
+          bosonVoucher.connect(assistant).callExternalContract(ethers.constants.AddressZero, calldata)
+        ).to.be.revertedWith(RevertReasons.INVALID_ADDRESS);
+      });
+
+      it("Caller is not the contract owner", async function () {
+        await expect(
+          bosonVoucher.connect(rando).callExternalContract(mockSimpleContract.address, calldata)
+        ).to.be.revertedWith(RevertReasons.OWNABLE_NOT_OWNER);
+      });
+
+      it("External call reverts", async function () {
```

Comment:
Please add a test, where `_to` is an EOA, which should also fail.

Reply:
done